hibernate-agroal/src/main/java/org/hibernate/agroal/internal/AgroalConnectionProvider.java

```diff
@@ -76,7 +76,9 @@ private static <T> void copyProperty(Map<String, Object> properties, String key,
 
 	@Override
 	public void configure(Map<String, Object> props) throws HibernateException {
-		LOGGER.debug( "Configuring Agroal" );
+		if ( LOGGER.isTraceEnabled() ) {
+			LOGGER.trace( "Configuring Agroal" );
+		}
```

Comment:
I guess I don't see how there's any value to this change.

Reply:
@sebersole suggested it